### PR TITLE
Russian language update

### PR DESCRIPTION
**Pull request by CalibriLight**
If you need to contact me: CalibriLightYT#8184

### CODE REVIEW

**Review by CalibriLight — COMMENTED** [review]
Finally, ive done everything, gonna pr

**Comment by mty22** [comment]
Thanks! I fixed up the merge conflict. Just need another native Russian speaker to review your changes. :)

**Comment by CalibriLight** [comment]
:)

**Comment by CalibriLight** [comment]
Finally, everything was done :)

**Comment by CalibriLight** [comment]
101% translate

**Comment by CalibriLight** [comment]
:)

**Comment by CalibriLight** [comment]
I bet everything is translated :)

**Comment by CalibriLight** [comment]
`####`
`#`
`####`
`#`
`####`

**Comment by CalibriLight** [comment]
e

File: ru.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ generic.accuracy Точность
 player.waiting Ожидание игроков
 player.died Вы умерли
 player.killed Убиты
-player.killstreak {0} on a {1} Kill Streak
+player.killstreak {0} сделал {1} убийств подряд!
```
Comment (mty22) [inline ru.js:64]: Can you revert this one back to English please

**Review by mty22 — CHANGES_REQUESTED** [review]
Just need you to change this one back to English.

Also, do you have any native Russian speaker friends who can review?

**Comment by mty22** [comment]
is this abandoned?

File: ru.js (hunk 1 above)
Comment (PrinceXawkwa) [reply ru.js:64]: @@ -61.7 +61.7 @@ generic.accuracy Accuracy
player.waiting Waiting for players
player.died You are dead
player.killed Killed
player.killstreak {0} on a {1} Kill Streak
player.killstreak {0} made {1} kills in a row!

**Comment by mty22** [comment]
> > Thanks! I fixed up the merge conflict. Just need another native Russian speaker to review your changes. :)
> 
> Hey, I can be the one.

go for it!

File: ru.js [hunk 2]
```diff
@@ -84,8 +84,8 @@ settings.none Настройки не найдены
 settings.backup Резервные настройки
 settings.controls.change Управление
 settings.controls.press Нажмите любую клавишу
-settings.controls.leaderboard InGame Leaderboard Trigger Type
-settings.controls.voicechat Voicechat Trigger Type
+settings.controls.leaderboard Триггер списка лидеров
```
Comment (Loreos7) [inline ru.js:87]:
```suggestion
settings.controls.leaderboard Способ открыть список лидеров
```

File: ru.js [hunk 3]
```diff
@@ -84,8 +84,8 @@ settings.none Настройки не найдены
...
+settings.controls.voicechat Триггер голосового чата
```
Comment (Loreos7) [inline ru.js:88]:
```suggestion
settings.controls.voicechat Способ активации голосового чата
```

File: ru.js [hunk 4]
```diff
@@ -94,7 +94,7 @@ settings.network.netRerouting Автоматическая маршрутиза
 settings.network.netRate Частота обновления (Hz)
 settings.network.lagComp Компенсация задержки
 settings.network.optNet Optimized Networking
-settings.network.quickplay Force Quickplay
+settings.network.quickplay Принужденный квикплей
```
Comment (Loreos7) [inline ru.js:97]:
```suggestion
settings.network.quickplay Быстрое подключение
```

File: ru.js [hunk 5]
```diff
@@ -649,12 +649,12 @@ windows.secondary.unlocked Разблокировано
 windows.secondary.customOnly Только для Пользовательских Игр
 windows.secondary.premiumOnly Только для Премиум-пользователей
 windows.secondary.req Требуется уровень {0}
-windows.skins.unlocked Unlocked
-windows.skins.customOnly Customs Only
-windows.skins.premiumOnly Premium Only
+windows.skins.unlocked Заблокировано
```
Comment (Loreos7) [inline ru.js:652]:
```suggestion
windows.skins.unlocked Разблокировано
```
???

File: ru.js [hunk 6]
```diff
@@ -649,12 +649,12 @@ windows.secondary.unlocked Разблокировано
...
+windows.skins.customOnly Только Кастом игры
```
Comment (Loreos7) [inline ru.js:653]:
```suggestion
windows.skins.customOnly Только для Пользовательских Игр
```

File: ru.js [hunk 7]
```diff
@@ -901,15 +901,15 @@ stats.airdrops.stolen Похищенные аирдропы
 settings.quality.flapAnim Анимация снайперской крышки
 chall.name.hs Сделано {0} убийств в голову {1}
 chall.name.ns Сделано {0} ноускопов {1}
-chall.name.ls Get {0} legshots {1}
-chall.name.tmk Get {0} throwing knife kills {1}
-chall.name.fb Get {0} first bloods {1}
+chall.name.ls Сделано {0} киллов в ногу {1}
```
Comment (Loreos7) [inline ru.js:904]:
```suggestion
chall.name.ls Сделано {0} убийств в ногу {1}
```

File: ru.js [hunk 8]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
-commands.forfeit.ended Forfeiting Game
-commands.forfeit.ended.other Other Team Forfeited
-server.message.join {0} joined the game
-server.message.left {0} left the game
-server.message.kicked {0} was kicked
-server.message.banned {0} was banned
-server.message.votekicked {0} was vote kicked
-server.message.arrested {0} was arrested by the K.P.D
+event.doubleXP В этом матче вы будете получать 2x больше опыта на класс
+ranked.join Вы присоединились к ранговой игре. За преждевременный выход до конца матча вы получите временный бан.
```
Comment (Loreos7) [inline ru.js:924]:
```suggestion
ranked.join Вы присоединились к рейтинговой игре. За преждевременный выход из матча вы получите временный бан.
```

File: ru.js [hunk 9]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+ranked.progress Игра продолжается.
+ranked.team Пожалуйста, присоединитесь к команде
+ranked.switch Вы не можете менять класс/команду во время игры
+ranked.spectate Вы не можете играть за наблюдателя в ранговом матче
```
Comment (Loreos7) [inline ru.js:928]:
```suggestion
ranked.spectate Вы не можете играть за наблюдателя в рейтинговом матче
```

File: ru.js [hunk 10]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+ranked.login Войдите в аккаунт для начала
```
Comment (Loreos7) [inline ru.js:929]:
```suggestion
ranked.login Войдите, чтобы начать
```

File: ru.js [hunk 11]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+ranked.sync Не удается синхронизировать данные
+ranked.team.spawn Присоединитесь к команде для начала
```
Comment (Loreos7) [inline ru.js:931]:
```suggestion
ranked.team.spawn Присоединитесь к команде, чтобы начать
```

File: ru.js [hunk 12]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+ranked.class.spawn Выберите класс для начала
```
Comment (Loreos7) [inline ru.js:932]:
```suggestion
ranked.class.spawn Выберите класс, чтобы начать
```

File: ru.js [hunk 13]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+modes.round.win Ваша команда победила этот раунд!
```
Comment (Loreos7) [inline ru.js:933]:
```suggestion
modes.round.win Ваша команда взяла этот раунд!
```

File: ru.js [hunk 14]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+modes.round.loss Ваша команда проиграла этот раунд
```
Comment (Loreos7) [inline ru.js:934]:
```suggestion
modes.round.loss Этот раунд больше не выиграть
```

File: ru.js [hunk 15]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+modes.round.switch Смена сторон. Приготовьтесь к следующему раунду
+modes.trade.success Успешный обмен!
+modes.prophunt.limit [Prop Hunt] Вы больше не можете изменить свой облик!
+modes.prophunt.left [Prop Hunt] Вы можете изменить облик {0} раз
```
Comment (Loreos7) [inline ru.js:938]:
```suggestion
modes.prophunt.left [Prop Hunt] Вы можете изменить облик ещё {0} раз
```

File: ru.js [hunk 16]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+commands.unlimited.off.other Бесконечные патроны выключены для {0}
+commands.spraybypass.on Бесконечные патроны включены
+commands.spraybypass.off Бесконечные патроны выключены
+commands.spraybypass.on.other Бесконечные патроны включены для {0}
+commands.spraybypass.off.other Бесконечные патроны выключены для {0}
+commands.suspect.invalid Некорректная информация о подозреваемом
+commands.suspect.menu Подозрительный игрок в меню
+commands.suspect.left Подозрительный игрок вышел из игры
+commands.suspect.found Обнаружен подозрительный игрок. Наблюдатель: {0}, Появился: {1}, Имя: {2}
+commands.forfeit.next Нужно сыграть хотя-бы 1 раунд
```
Comment (Loreos7) [inline ru.js:969]:
```suggestion
commands.forfeit.next Нужно сыграть хотя бы 1 раунд
```

File: ru.js [hunk 17]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+commands.forfeit.wait Дождитесь до начала матча, чтобы сдаться
```
Comment (Loreos7) [inline ru.js:970]:
```suggestion
commands.forfeit.wait Дождитесь начала матча, чтобы сдаться
```

File: ru.js [hunk 18]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+commands.smite.off Бесконечный урон выключен
+commands.smite.on.other Бесконечный урон включен для {0}
+commands.smite.off.other Бесконечный урон выключен для {0}
+commands.perk Выдано {0} {1}
+commands.maxperk Выдано {0} MAX {1}
+commands.unlimited.on Бесконечные патроны включены
+commands.unlimited.off Бесконечные патроны выключены
+commands.unlimited.on.other Бесконечные патроны включены для {0}
+commands.unlimited.off.other Бесконечные патроны выключены для {0}
+commands.spraybypass.on Бесконечные патроны включены
```
Comment (Loreos7) [inline ru.js:961]:
```suggestion
commands.spraybypass.on Бесконечный спрей включён
```

File: ru.js [hunk 19]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+commands.spraybypass.off Бесконечные патроны выключены
```
Comment (Loreos7) [inline ru.js:962]:
```suggestion
commands.spraybypass.off Бесконечный спрей выключен
```
А читать умеем?

File: ru.js [hunk 20]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+commands.spraybypass.on.other Бесконечные патроны включены для {0}
```
Comment (Loreos7) [inline ru.js:963]:
```suggestion
commands.spraybypass.on.other Бесконечный спрей включён для {0}
```

File: ru.js [hunk 21]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+commands.spraybypass.off.other Бесконечные патроны выключены для {0}
```
Comment (Loreos7) [inline ru.js:964]:
```suggestion
commands.spraybypass.off.other Бесконечный спрей выключен для {0}
```

File: ru.js [hunk 22]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+modes.trade.success Успешный обмен!
+modes.prophunt.limit [Prop Hunt] Вы больше не можете изменить свой облик!
+modes.prophunt.left [Prop Hunt] Вы можете изменить облик {0} раз
+modes.prophunt.rotation.on [Prop Hunt] Вращение включено
+modes.prophunt.rotation.off [Prop Hunt] Вращение выключено
+modes.prophunt.rotation.reset [Prop Hunt] Сброс вращения.
+commands.login Войдите, чтобы использовать эту команду
+commands.connected {0} подключено, {1} в меню
+commands.host {0} является хостом.
+commands.noclip.on Ноклип включен
```
Comment (Loreos7) [inline ru.js:945]:
```suggestion
commands.noclip.on Noclip включен
```
Функции, у которых нет нормального перевода, не стоит транслитировать

File: ru.js [hunk 23]
```diff
@@ -920,63 +920,63 @@ chall.name.dmg Нанесено {0} урона {1}
...
+commands.noclip.off Ноклип выключен
```
Comment (Loreos7) [inline ru.js:946]:
```suggestion
commands.noclip.off Noclip выключен
```

**Review by Loreos7 — CHANGES_REQUESTED** [review]

File: ru.js (hunk 22 above)
Comment (CalibriLight) [reply ru.js:945]: мда, я дурачок :/

**Comment by mty22** [comment]
Treating this one as abandoned. Please review https://github.com/mty22/krunker.io-translations/pull/493

**Comment by mty22** [comment]
@CalibriLight see comment above, I'll send you a reasonable amount of KR if you can review https://github.com/mty22/krunker.io-translations/pull/493 please.

**Pull request closed without merging**